pkg/gather/clusterconfig/certificate_signing_requests.go

```diff
@@ -98,8 +101,8 @@ func FromCSRs(requests *certificatesv1api.CertificateSigningRequestList) *CSRs {
 
 func (c *CSRs) Anonymize() *CSRs {
 	res := &CSRs{}
-	for _, r := range c.Requests {
-		af := anonymizeCSR(&r)
+	for _, r := range c.Requests { //nolint: gocritic
+		af := anonymizeCSR(&r) //nolint: gosec
```

Comment:
Maybe we could use indexing? As SO told me https://stackoverflow.com/questions/62446118/implicit-memory-aliasing-in-for-loop